src/main/java/ui/listpanel/ListPanelCard.java

```diff
@@ -124,8 +130,18 @@ private static Node layoutEvents(Model model, TurboIssue issue,
         result.setSpacing(3);
         VBox.setMargin(result, new Insets(3, 0, 0, 0));
 
-        // Events
+        // Label update events
+        List<TurboIssueEvent> labelUpdateEvents =
+                        events.stream()
+                        .filter(e -> e.isLabelUpdateEvent())
```

Comment:
This one doesn't work since `e` is the object on which `isLabelUpdateEvent` is called, not technically its argument and also `isLabelUpdateEvent` is not static.